core/shared/platform/common/posix/posix_thread.c

```diff
@@ -492,6 +492,9 @@ destroy_stack_guard_pages()
 }
 #endif /* end of WASM_DISABLE_STACK_HW_BOUND_CHECK == 0 */
 
+#if defined(__GNUC__)
+__attribute__((no_sanitize_address))
```

Comment:
could we please have a comment in the code explaining why do we disable sanitizer for this function?